redi-connect-front/src/components/organisms/ApplyForm.tsx

```diff
@@ -54,7 +54,7 @@ interface Props {
   mentorId: string
 }
 
-export const ConnectionRequestForm = ({ mentorId }: Props) => {
+const ApplyForm = ({ mentorId }: Props) => {
```

Comment:
Let's keep the level of explicitness. The verb 'apply' is a good choice - let's name this something like `MentorshipApplyForm`